snapcraft/extensions/gnome.py

```diff
@@ -0,0 +1,212 @@
+# -*- Mode:Python; indent-tabs-mode:nil; tab-width:4 -*-
+#
+# Copyright (C) 2022 Canonical Ltd
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License version 3 as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+"""Generic GNOME extension to support core22 and onwards."""
+
+
+from typing import Any, Dict, Optional, Tuple
+
+from overrides import overrides
+
+from .extension import Extension, get_extensions_data_dir
+
+_PLATFORM_SNAP = dict(core22="gnome-42-2204")
+_SDK_SNAP = dict(core22="gnome-42-2204-sdk")
```

Comment:
Any special reason for not using literal dictionaries instead of `dict()`?

Reply:
copy paste, this can change :-)